templates/resources/consul_cluster.md.tmpl

```diff
@@ -9,6 +9,8 @@ description: |-
 
 -> **Note:** The `primary_link` attribute is related to federation, a feature that is currently in private beta.
 
+-> **Note:** The `auto_hvn_to_hvn_peering` attribute is related to federation, a feature that is currently in private beta.
```

Comment:
This could be merged with above:
```suggestion
-> **Note:** The `auto_hvn_to_hvn_peering` and `primary_link` attributes are related to federation, a feature that is currently in private beta.
```